test/javascripts/acceptance/assigned-topic-test.js

```diff
@@ -123,6 +130,32 @@ acceptance("Discourse Assign | Assigned topic", function (needs) {
       "shows reassign dropdown at the bottom of the topic"
     );
   });
+
+  test("Shows assignement notification", async (assert) => {
+    updateCurrentUser({ can_assign: true });
+
+    await visit("/u/eviltrout/notifications");
+
+    debugger;
+
+    let notification = query("section.user-content ul.notifications li.item.notification")[0];
```

Comment:
This is the showstopper you were hitting: 😃 

```suggestion
    let notification = query("section.user-content ul.notifications li.item.notification");
```